### PR TITLE
Use consistent file naming of documentation translations.

### LINKED ISSUE #1331: Documentation: Language suffix inconsistently with "-" or "_"
There is this instance of
```
$ ls -l  docs/src/getting-started/running-linuxcnc_es.txt docs/src/getting-started/running-linuxcnc-es.txt
-rw-r--r-- 1 steffen steffen 2050 Sep 23 18:27 docs/src/getting-started/running-linuxcnc-es.txt
-rw-r--r-- 1 steffen steffen 7565 Sep 23 18:27 docs/src/getting-started/running-linuxcnc_es.txt
```
which I read as somehow everyone is confused about what to use. My suggestion is to use the underscore consistently, since many more files adhere to that scheme:
```
steffen@wurzel:~/Github/linuxcnc$ find . -name "*_es.*" | grep -v debian|wc -l
314
$ find docs/ -name "*-es*"
docs/src/getting-started/running-linuxcnc-es.txt
docs/src/getting-started/updating-linuxcnc-es.txt
docs/src/getting-started/about-linuxcnc-es.txt
docs/src/ladder/images/axis_cl-estop.png
docs/src/config/stepper-quickstart-es.txt
docs/src/config/images/switch-no-parallel-es.png
docs/src/config/images/switch-nc-series-es.png
```
Similar for french

```
steffen@wurzel:~/Github/linuxcnc$ find . -name "*_fr.*" | grep -v debian|wc -l
249
steffen@wurzel:~/Github/linuxcnc$ find . -name "*-fr.*" | grep -v debian|wc -l
9
```
And somehow I seem to miss something for the chinese
```
steffen@wurzel:~/Github/linuxcnc$ find . -name "*_cn.*" | grep -v debian|wc -l
5
steffen@wurzel:~/Github/linuxcnc$ find . -name "*-cn.*" | grep -v debian|wc -l
17
```
since the *_cn.pdf is not built for me that is expected by the .desktop file.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**
Use '_XX' instead of '-XX' consistently, as _XX was already used by the
majority of translated files.

If there were several translations of the same file, the '-XX' one was deleted
without considering which translation is most up to date.

This change make the build rules for handling translations using po4a easier
to write, as there is no longer any need for special case handling.

Fixes #1331

### CODE REVIEW

**Comment by petterreinholdtsen** [comment]
This pull request will need an update after the build problem workaround in #1345 is merged.

**Comment by SebKuzminsky** [comment]
Oops, I reimplemented parts of this PR independently and pushed to master.  This PR does a couple of extra things that I did not do (rename some SVGs to follow the "_<lang>" convention, possibly other things).  Please rebase and I'll take a look at what's left here.

**Comment by petterreinholdtsen** [comment]
[Sebastian Kuzminsky]
> Oops, I reimplemented parts of this PR independently and pushed to
> master.  This PR does a couple of extra things that I did not do
> (rename some SVGs to follow the "_<lang>" convention, possibly other
> things).  Please rebase and I'll take a look at what's left here.

Done.

--
Happy hacking
Petter Reinholdtsen

File: docs/src/index_cn.tmpl [hunk 1]
```diff
@@ -72,8 +72,8 @@ function change(el)
 <li><a onclick="return toggle('sec1')"><img id="sec1_image" src="plus.png" style="border:0;margin-right:5px;vertical-align:middle;" />Configuration Wizards</a>
 	<div id="sec1" style="display:none;">
 		<UL>
-			<LI><A class="tooltips" HREF="config/stepconf-cn.html">Stepper Configuration Wizard<span>For Parallel Port Stepper Machine</span></A>
-			<LI><A class="tooltips" HREF="config/pncconf-cn.html">Mesa Configuration Wizard<span>For Mesa Servo &amp; Stepper Machines</span></A>
+			<LI><A class="tooltips" HREF="config/stepconf_cn.html">Stepper Configuration Wizard<span>For Parallel Port Stepper Machine</span></A>
+			<LI><A class="tooltips" HREF="config/pncconf_cn.html">Mesa Configuration Wizard<span>For Mesa Servo &amp; Stepper Machines</span></A>
```
Comment (SebKuzminsky) [inline docs/src/index_cn.tmpl:76]: All four of these files (`{step,pnc}conf{-,_}cn`) don't exist - just another issue with the Chinese translation.  For now the links should point to the english (untranslated) versions.

File: debian/extras/linuxcnc.files [hunk 2]
```diff
@@ -9,10 +9,10 @@ usr/share/icons/hicolor/scalable/apps/linuxcnc-logo.svg
...
-usr/share/icons/hicolor/scalable/apps/linuxcnc-en.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-es.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-fr.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-vi.svg
-usr/share/icons/hicolor/scalable/apps/linuxcnc-cn.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_en.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_es.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_fr.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_vi.svg
+usr/share/icons/hicolor/scalable/apps/linuxcnc_cn.svg
```
Comment (SebKuzminsky) [inline debian/extras/linuxcnc.files:16]: These renames require that you also update the Icon filenames in the .desktop files in `debian/extras/usr/share/applications/`.

**Review by SebKuzminsky — CHANGES_REQUESTED** [review]
Thanks for working on this, it's very close to ready.

**Review by smoe — COMMENTED** [review]
Done.

**Comment by petterreinholdtsen** [comment]
I've updated the ones you discovered, and renamed a few more files according to the new notation.  Hope I got it all now.

**Comment by smoe** [comment]
Well. The .txt -> .adoc rename PR https://github.com/LinuxCNC/linuxcnc/pull/1344 from what I see has these already covered. These rebases and manual checks meant far more work than the original effort to perform the rename in a for loop and sed on the includes. This was a whole morning that I cannot invest again. I just hope this wasn't the same for you now, @petterreinholdtsen .

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> Well. The .txt -> .adoc rename PR
> https://github.com/LinuxCNC/linuxcnc/pull/1344 from what I see has
> these already covered. These rebases and manual checks meant far more
> work than the original effort to perform the rename in a for loop and
> sed on the includes. This was a whole morning that I cannot invest
> again. I just hope this wasn't the same for you now,
> @petterreinholdtsen .

I am perfectly happy with the txt->adoc rename to happen first, and can
rebase these changes quite quickly.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by SebKuzminsky** [comment]
> These rebases and manual checks meant far more work than the original effort to perform the rename in a for loop and sed on the includes. This was a whole morning that I cannot invest again.

I'm sorry if the order I merged things in caused extra work for you, that was not my intent.  :-(

**Comment by SebKuzminsky** [comment]
@petterreinholdtsen, thank you for these updates.

**Pull request merged**